WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -50,7 +50,10 @@
           <ul class="dropdown-menu" role="menu">
             <li><a href="/results/events.php"><%= icon('signal', class: 'fa-fw fa-rotate-90') %> <%= t '.rankings' %></a></li>
             <li><a href="/results/regions.php"><%= icon('trophy', class: 'fa-fw') %> <%= t '.records' %></a></li>
-            <li><a href="<%= persons_path %>"><%= icon('user', class: 'fa-fw') %> <%= t '.persons' %></a></li>
+            <li><a href="<%= persons_path %>"><%= icon('users', class: 'fa-fw') %> <%= t '.persons' %></a></li>
```

Comment:
I think this `fa-fw` class is redundant and there's not reason to keep it. Instead I'd suggest changing [this line](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/assets/stylesheets/navbar-static-top.scss#L8) to use just `fa` instead of `fa-fw`.